web/tests/test_quiz_admin.py

```
@@ -0,0 +1,135 @@
+
+        if "adminform" in response.context:
+            print("Errors:", response.context["adminform"].form.errors)
+        else:
+            print("No adminform found in response context")
+        
+        
+
+        self.assertEqual(response.status_code, 200)  # Ensure form is processed
+        self.assertTrue(QuizQuestion.objects.filter(question_text="What is 2 + 2?").exists())
+
+    def test_quiz_question_admin_add(self):
```

Ruff (0.8.2)

Redefinition of unused `test_quiz_question_admin_add` from line 76

(F811)